src/styles/layout/index.md.njk

```diff
@@ -123,3 +123,18 @@ The desktop specific classes also allow you to set the width of the tablet break
 ### Nested grids
 
 {{ example({group: "styles", item: "layout", example: "nested", html: true, open: true, size: "m"}) }}
+
+## Display override classes
+
+If you need to override the default way that certain elements display on the page, you can use the display override classes.
+
+There are override classes for each of the following CSS `display` properties:
+
+- inline -  `.govuk-!-display-inline` 
+- inline-block -  `.govuk-!-display-inline-block` 
+- block -  `.govuk-!-display-block` 
+- none - `.govuk-!-display-none`
```

Comment:
Would suggest removing the leading `.` as you don't want it when you're using the class (only when authoring) and it's consistent with how we refer to classes elsewhere.

```suggestion
- inline -  `govuk-!-display-inline` 
- inline-block -  `govuk-!-display-inline-block` 
- block -  `govuk-!-display-block` 
- none - `govuk-!-display-none`
```